### PR TITLE
Feature/poster flip

### PR DESCRIPTION
**Pull request by emmacbd**
## What does this PR do?
Adds poster flip feature to home page posters outside of carousel, adds font styling to home page.

## Where should the reviewer start?
Poster.js or poster.css

## How should this be manually tested?
Run npm start

### CODE REVIEW

File: src/Poster-Container.js [hunk 1]
```diff
@@ -9,7 +9,9 @@ const PosterContainer = ({movies, showMovieSummary}) => {
 
       <Poster
         imagePath={movie.poster_path}
+        backdropPath={movie.backdrop_path}
         title={movie.title}
+        rating={Math.round(movie.average_rating)}
```
Comment (sted1994) [inline src/Poster-Container.js:14]: Maybe we can see what toFixed() looks like here. This looks good but we might want to keep keep the first decimal place.

File: src/Poster.js [hunk 2]
```diff
@@ -2,11 +2,20 @@ import React from "react";
...
-      <div  id={id} onClick={(event) => showMovieSummary(event)} className="poster">
-        <img  id={id} className="poster-image" src={imagePath} alt={`movie poster for ${title}`}/>
-        <h3 id={id}>{title}</h3>
+      <div id={id} onClick={(event) => showMovieSummary(event)} className="poster">
+       <div className="poster-inner">
+        <div className="poster-front">
+          <img  id={id} className="poster-image" src={imagePath} alt={`movie poster for ${title}`}/>
+          <h3 id={id}>{title}</h3>
+        </div>
+        <div className="poster-back">
```
Comment (sted1994) [inline src/Poster.js:13]: This was a really good idea to include a "back" to the cards to show a bit more.

File: src/css/poster.css [hunk 3]
```diff
@@ -1,9 +1,46 @@
...
 }
 
-.poster-image {
-  width: 100%
+.poster-image, .backdrop-image {
+  width: 100%;
+  cursor: pointer;
+}
+
+/* This container is needed to position the front and back side */
```
Comment (sted1994) [inline src/css/poster.css:14]: Thank you for leaving the comments in here.

**Review by sted1994 — APPROVED** [review]
Look good! Thank you for getting this done.

File: src/Poster.js [hunk 4]
```diff
@@ -2,11 +2,20 @@ import React from "react";
 import "./css/poster.css";
 
 
-const Poster = ({imagePath, title, id, showMovieSummary}) => {
+const Poster = ({imagePath, backdropPath, title, rating, id, showMovieSummary}) => {
     return (
-      <div  id={id} onClick={(event) => showMovieSummary(event)} className="poster">
-        <img  id={id} className="poster-image" src={imagePath} alt={`movie poster for ${title}`}/>
-        <h3 id={id}>{title}</h3>
+      <div id={id} onClick={(event) => showMovieSummary(event)} className="poster">
```
Comment (michaelputnam67) [inline src/Poster.js:7]: We may want to refactor this at some point to eliminate some of the divs.

**Review by michaelputnam67 — COMMENTED** [review]
Looks awesome! way to figure it out.

File: src/Poster.js [hunk 5]
```diff
@@ -2,11 +2,20 @@ import React from "react";
...
+       <div className="poster-inner">
+        <div className="poster-front">
+          <img  id={id} className="poster-image" src={imagePath} alt={`movie poster for ${title}`}/>
+          <h3 id={id}>{title}</h3>
```
Comment (kaylagordon) [inline src/Poster.js:11]: Do the `img` and `h3` elements need the `id`?

File: src/css/app.css [hunk 6]
```diff
@@ -1,5 +1,7 @@
 main {
 	margin: 0;
+  margin-left: -10px;
 	background: black;
 	width: 100vw;
+  font-family: 'Teko', sans-serif;
```
Comment (kaylagordon) [inline src/css/app.css:6]: indentation

File: src/css/carousel.css [hunk 7]
```diff
@@ -3,12 +3,15 @@
...
   margin-top: 20px;
+  margin-bottom: 7%;
   overflow-x: scroll;
   width: 80vw;
 }
 
 .top-rated {
   color: white;
+  font-size: 100pt;
+
```
Comment (kaylagordon) [inline src/css/carousel.css:14]: remove empty line

File: src/css/nav.css [hunk 8]
```diff
@@ -5,11 +5,13 @@
...
+  background-color: #414141;
 }
 
 .logo {
 	height: 100%;
 }
 
-
+.logo:hover {
+	cursor: pointer;
```
Comment (kaylagordon) [inline src/css/nav.css:16]: indentation

File: src/Poster-Container.js [hunk 9]
```diff
@@ -9,7 +9,9 @@ const PosterContainer = ({movies, showMovieSummary}) => {
```
Comment (kaylagordon) [inline src/Poster-Container.js:9]: remove empty line

**Pull request merged**